packages/trpc/server/routers/viewer/workflows/getVerifiedEmails.handler.ts

```diff
@@ -17,7 +15,7 @@ type GetVerifiedEmailsOptions = {
 export const getVerifiedEmailsHandler = async ({ ctx, input }: GetVerifiedEmailsOptions) => {
   const { user } = ctx;
   const { teamId } = input;
-  let verifiedEmails: VerifiedEmail[] = [];
+  let verifiedEmails: string[] = [user.email];
```

Comment:
user email should show as automatically verified

we only need the emails, so no need to return the whole `VerifiedEmail` object

Reply:
Could it be that the logged in user hasn't verified his email and setting up the workflow?

Would it be okay to consider the email as verified in that case.